### ISSUE TITLE
N measurement/sensor/param

### ISSUE THREAD

**Issue by castelao** [OceanGlidersCommunity/OG-format-user-manual#60]
Raised by @vturpin at #35 

Is it worth having N_MEASUREMENT, N_SENSOR, and N_PARAM as index values for each one? I removed for now from the example since it is not explicitly defined in the document. If it is decided on using it, remember to update in the example.

**Comment by castelao** [comment]
Adding some info. If maintained it would include something like

```
...
 N_MEASUREMENTS = 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24 ;

 N_SENSOR = 0, 1, 2, 3, 4 ;

 N_PARAM = 0, 1, 2, 3, 4 ;
...
```

**Comment by glidermann** [comment]
Yes, thanks @vturpin. I would like feedback from @tcarval or others who use this in EGO. I understand the intention to have all relevant sensors listed somehow for each parameter (your second example above). Leaving it empty may be easy to manage but then what's the point?

**Comment by vturpin** [comment]
Closed since we agree on #103
